code/datums/skills/bay_skills.dm

```diff
@@ -0,0 +1,497 @@
+/datum/skill_list_bay
+	var/name = "Default"
+	var/unarmed = 0 // Increases unarmed damage.
+	var/melee = 0 // Increases weapon damage + block chance with shields.
+	var/engineering = 0 // Affects speed of do_afters with engi tools.
+	var/salvaging = 0 // Affects drop-rates of salvage.
+	var/smithing = 0 // Smithing. Affects quality of smithed weapons.
+	var/chemistry = 0 // If too low - you'll have a delay before using chemistry machines.
+	var/medical = 0 // First Aid. Affects speed of applying sutures/gauzes and fixing wounds.
+	var/surgery = 0 // Surgery Steps speed.
+	var/crafting = 0 // How fast you craft items.
+	var/culinary = 0 // How fast you craft food.
+	var/science = 0 // If too low - you'll have a delay before using some scientific machines.
+
+/datum/skill_list_bay/proc/getRating(rating)
+	return vars[rating]
+
+/datum/skill_list_bay/proc/getList()
+	return list("unarmed" = unarmed,\
+		"melee" = melee,\
+		"engineering" = engineering,\
+		"salvaging" = salvaging,\
+		"smithing" = smithing,\
+		"chemistry" = chemistry,\
+		"medical" = medical,\
+		"surgery" = surgery,\
+		"crafting" = crafting,\
+		"culinary" = culinary,\
+		"science" = science)
```

Comment:
```suggestion
/datum/skill_list_bay/proc/getList()
	return list(
		"unarmed" = unarmed,
		"melee" = melee,
		"engineering" = engineering,
		"salvaging" = salvaging,
		"smithing" = smithing,
		"chemistry" = chemistry,
		"medical" = medical,
		"surgery" = surgery,
		"crafting" = crafting,
		"culinary" = culinary,
		"science" = science,
		)
```
* Trailing comma, entries on their own lines and removal of superfluous escapes.
* This also needs to be using defines, and the name of the proc is awfully uninformative.